### PR TITLE
GUI: add additional ws message for debugging (#810)

### LINKED ISSUE #810: BUG-Module Python model is not sent via websocket
Let this example : <https://github.com/bengaid/velib_taipy_chalkit>

Clone it to get it. Run it in the latest Taipy / Chalk'it setup.

At browser side, module `velib_real_time_page` variables does not exist : 

image

Thus, it is not possible to perform bindings with Chalk'it variables.

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Resolve #810

Don't directly resolve the issue but might be necessary to detect potential problems

### CODE REVIEW

**Review by FredLL-Avaiga — DISMISSED** [review]
Always good to have more info

File: frontend/taipy-gui/base/src/utils.ts [hunk 1]
```diff
@@ -19,16 +19,19 @@ export const initSocket = (socket: Socket, appManager: TaipyApp) => {
     });
     // Send a request to get App ID to verify that the app has not been reloaded
     socket.io.on("reconnect", () => {
+        console.log("WebSocket reconnected")
         sendWsMessage(socket, "AID", "reconnect", appManager.appId, appManager.clientId, appManager.context);
     });
     // try to reconnect on connect_error
-    socket.on("connect_error", () => {
+    socket.on("connect_error", (err) => {
+        console.log("WebSocket connect_error: ", err);
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/base/src/utils.ts]:
```suggestion
        console.log("Error connecting WebSocket: ", err);
```

**Review by FabienLelaquais — DISMISSED** [review]
👌

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**